warehouse/cli/search/reindex.py

```diff
@@ -57,6 +57,8 @@ def reindex(config, **kwargs):
     """
     client = config.registry["elasticsearch.client"]
     db = Session(bind=config.registry["sqlalchemy.engine"])
+    number_of_replicas = config.registry.get("elasticsearch.replicas", 0)
+    refresh_interval = config.registry.get("elasticsearch.interval", "5s")
```

Comment:
Currently it only gets updated during bulk indexing, however once we kill legacy PyPI then I plan on making it so that bulk indexing is only a rare thing and most updates happen as updates happen to the source data.

That being said, even when we are doing incremental updates as the source data changes, we don't really need super realtime data. If something shows up within a few minutes that should be more than quick enough.